client/tests/integration/multisig.rs

```diff
@@ -0,0 +1,170 @@
+use std::{
+    collections::{BTreeMap, BTreeSet},
+    str::FromStr,
+};
+
+use eyre::Result;
+use iroha::{
+    client::{self, ClientQueryError},
+    crypto::KeyPair,
+    data_model::{
+        account::NewAccount,
+        prelude::*,
+        transaction::{TransactionBuilder, WasmSmartContract},
+    },
+};
+use iroha_data_model::parameter::SmartContractParameter;
+use nonzero_ext::nonzero;
+use serde::{Deserialize, Serialize};
+use test_network::*;
+use test_samples::{gen_account_in, ALICE_ID};
+
+/// Mirror trigger args for `multisig_register`
+#[derive(Serialize, Deserialize)]
+struct MultisigRegisterArgs {
+    // Account id of multisig account should be manually checked to not have corresponding private key (or having master key is ok)
+    account: NewAccount,
+    // List of accounts responsible for handling multisig account
+    signatories: BTreeSet<AccountId>,
+}
+
+/// Mirror trigger args for `multisig`
+#[derive(Serialize, Deserialize)]
+enum MultisigArgs {
+    /// Accept instruction and vote
+    Instruction(InstructionBox),
+    /// Accept vote for certain instruction
+    Vote(HashOf<InstructionBox>),
+}
```

Comment:
> and why not an executable but an instruction? 

Afaik currently we currently don't have ability to execute `Executable` from the `wasm` only one instruction at a time.

> Do we limit to one instruction at a time? Or does this just intend simplicity as a prototype?

Yes and yes, i want to make implementation simple. 
In case we want more sophisticated schemes we can accept `Vec<InstructionBox>`. 

This way user would be able to have smt like `[register by call trigger, execute trigger]` which is kinda workaround to execute arbitrary executable.